smithy-security/smithy
# Implement persistent findings storage - Part 4

To be merged after https://github.com/smithy-security/smithy/pull/494 and after publishing a new SDK version.

![image](https://github.com/user-attachments/assets/686c3d14-2940-4de0-b839-d35ff84b5fe3)

- Review https://github.com/smithy-security/saas/pull/806 also

---

Plugging everything together using a persistent storage on top of psql in the existing gosec workflow:
- updating to temporary version of SDK
- sample docker-compose pipeline

---

How to test:
- go inside `new-components`
- run `docker-compose up --build --force-recreate`
- check that everything builds fine
- check the logs from the final json-logger to output one vuln
- run `docker-compose down --rmi all` to shut everything down
